### DISCUSSION TITLE
no upgrade?

### DISCUSSION THREAD

**Discussion by DerMeso** [netbox-community/netbox#8989] [Q&A]
I am using the Official Upgrade method for Docker Netbox, yet it is not upgrading my container.

# Update the configuration files
git pull origin release

# Get the latest containers
docker-compose rm -fs netbox netbox-worker
docker-compose pull
docker-compose up -d netbox netbox-worker

**Comment by kkthxbye-code** [comment]
This discussion probably belongs in the netbox-docker repo instead. Also you haven't provided enough information for anyone to help you. What version are you trying to upgrade from/to?
